board_client/animations/checkerboard_pulse.py

```
@@ -1,34 +1,142 @@
 import uasyncio
 import math
+import random
+import utime
 
-from animations.utils import hsv_to_rgb, fast_sin
+from animations.utils import hsv_to_rgb, fast_sin, fast_cos, SIN_TABLE, COS_TABLE
```

Flake8 (7.2.0)

[error] 'animations.utils.fast_sin' imported but unused

(F401)